be/test/common/status_test.cpp

```diff
@@ -73,4 +73,10 @@ TEST_F(StatusTest, Error) {
     }
 }
 
+TEST_F(StatusTest /*unused*/, Format /*unused*/) {
+    // default
+    Status st;
```

Comment:
please add 3 tests：
1.  status == ok
2.  status == error && has stacktrace
3. status == error && do not have stacktrace

Reply:
Whether status contains stacktrace is determined by the preprocessor macro `ENABLE_STACKTRACE`. So for cases 2 and 3, I just need to test `status == error`, not care stacktrace?

Reply:
if not, please give me an example.